packages/cockroach/extra/register.py

```diff
@@ -278,21 +278,69 @@ def _init_cockroachdb_cluster(
     """
     @retrying.retry(
         wait_fixed=1000,
-        stop_max_attempt_number=10,
+        stop_max_attempt_number=60,
         retry_on_result=lambda x: x is False,
-        )
-    def _wait_for_pidfile(pid: int) -> bool:
+    )
+    def _wait_for_cluster_init() -> bool:
         """
-        Try and read the PID from `PID_FILE_PATH` and match it to the given `pid`.
-        If, after 10 seconds, the PIDs don't match, raise an exception.
+        CockroachDB Cluster initialization takes a certain amount of time
+        while the cluster ID and and node ID are written to the storage.
+
+        We wait 1 second to avoid having a huge amount of logs in the journal.
```

Comment:
I don't understand this argument. Maybe write "We wait 1 s instead of ..... to avoid" so that it's clearer what the train of thought was. Notably, I have commented on the 1 second timeout constant below.